bitbots_quintic_walk/include/bitbots_quintic_walk/walk_stabilizer.h

```diff
@@ -16,8 +17,10 @@ class WalkStabilizer : public bitbots_splines::AbstractStabilizer<WalkResponse>
  public:
   WalkStabilizer();
   virtual void reset();
-  virtual std::unique_ptr<bio_ik::BioIKKinematicsQueryOptions> stabilize(const WalkResponse &positions);
+  std::unique_ptr<bio_ik::BioIKKinematicsQueryOptions> stabilize(const WalkResponse &response, const ros::Duration &dt) override;
 
+ private:
+   control_toolbox::Pid pid_trunk_pitch_;
 };
 }
```

Comment:
a bit more out of scope of this MR but will enhance readability again add commend '// namespace bitbots_quintic_walk' to make end of namespaces more clear.

Reply:
done